### PR TITLE
Kafka Connect Snowflake sink

### PR DESCRIPTION
**Pull request by ftisiot**
# What changed, and why it matters

Created content for the Kafka Connect Snowflake sink connector.

I have a valid Snowflake trial, contact me if you want to test it and I can add you to it

### CODE REVIEW

File: docs/products/kafka/kafka-connect/howto/snowflake-sink.rst [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+Example: Create a Snowflake sink connector on a topic in Avro format
+--------------------------------------------------------------------
+
+The example creates an Snowflake sink connector with the following properties:
+
+* connector name: ``my_snowflake_sink``
+* source topics: ``test``
+* Snowflake database: ``testdb``
+* Snowflake schema: ``testschema``
+* Snowflake URL: ``https://XX0000.eu-central-1.snowflakecomputing.com:443``
```
Comment (anelook) [inline docs/products/kafka/kafka-connect/howto/snowflake-sink.rst:127]: In the end I used URL without https and the port. and it worked

File: docs/products/kafka/kafka-connect/howto/snowflake-sink.rst [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+* Snowflake user: ``testuser``
+* User private key:: 
+
+    XXXXXXXYYY
```
Comment (anelook) [inline docs/products/kafka/kafka-connect/howto/snowflake-sink.rst:131]: Should be one liner - without it JSON won't be valid :D

**Review by anelook — CHANGES_REQUESTED** [review]
Francesco, thanks a lot for putting all these details! This is not easy stuff and will be so helpful for our users. Both who know Snowflake and who don't.

This was my first time using Snowflake, so I struggled a bit (and asked you ton of questions). However, I think that all details that are relevant to Aiven connector are well written with lots of hints and explanations. The issues I had were relevant to Snowflake. And the file you have about preparing Snowflake helped actually a lot.

I spotted couple of funny typos - check them out. But the rest looks good to me.

One thought I had in mind - that we will need to keep an eye on Snowflake instructions, because their UI might change. and this is an extra stuff for us to support. But I feel that it is worth it because our users will appreciate that extra info.

File: docs/products/kafka/kafka-connect/howto/snowflake-sink.rst (hunk 1 above)
Comment (ftisiot) [reply docs/products/kafka/kafka-connect/howto/snowflake-sink.rst:127]: so, only `XX0000.eu-central-1.snowflakecomputing.com`?

**Review by anelook — APPROVED** [review]
Thanks for adding improvements based on the review. Looks good to me! Merging

**Pull request merged**